src/main/scala/com/thirdship/libunit/units/MetricTSUnit.scala

```diff
@@ -0,0 +1,52 @@
+package com.thirdship.libunit.units
+
+import com.thirdship.libunit.utils.{ExactString, FuzzyString, WordString}
+import com.thirdship.libunit.utils.Helpers._
+import com.thirdship.libunit._
+
+class MetricTSUnit(unit: String, parseList: List[FuzzyString]) {
```

Comment:
I am not entirely sure how this interfaces with the exiting classes. But as it is not a TSUnit, it should not be called one. 
Where does this nest in, for example:

```
val baseUnit = "m"
val conversionMap: Map[String, Conversion[Double, Double]] = Map(
    baseUnit -> 1.asScalarFn,
    "km" -> 1000.asScalarFn,
    "ft"   -> (12/39.370).asScalarFn,
    "in"   -> (1/39.370).asScalarFn,
    "mil"  -> 1609.344.asScalarFn
)
val compressedParseMap = Map(
    "m".i 	-> List("meter".w),
    "km".i 	-> List("kilometer".w),
    "ft".i 	-> List("feet".i, "foot".i, "'".e),
    "in".i 	-> List("inch".w, "\"".e),
    "mil".i -> List("mile".w)
)
val data = new ConvertibleTSUnitData(baseUnit, "Length", conversionMap, compressedParseMap)
```